.drone.yml

```diff
@@ -36,32 +36,31 @@ steps:
       - cd ../am-base
       - chmod u+x docker-entrypoint.sh
       - chmod u+x scripts/*
-      - sed -i'.tmp' -e 's/FROM amster/FROM '499815288658.dkr.ecr.eu-west-1.amazonaws.com'\\/id\\/amster/g' Dockerfile
-      - sed -i'.tmp' -e 's/FROM am-empty/FROM '499815288658.dkr.ecr.eu-west-1.amazonaws.com'\\/id\\/am-base/g' Dockerfile
+      - sed -i'.tmp' -e 's/FROM amster/FROM '499815288658.dkr.ecr.eu-west-1.amazonaws.com'\\/forgeops\\/amster-base/g' Dockerfile
+      - sed -i'.tmp' -e 's/FROM am-empty/FROM '499815288658.dkr.ecr.eu-west-1.amazonaws.com'\\/forgeops\\/am-base/g' Dockerfile
     when:
       event: push
       branch: master
 
   - name: publish-am-empty
     image: plugins/ecr
     settings:
-      repo: id/am-base
+      repo: forgeops/am-base
       registry: 499815288658.dkr.ecr.eu-west-1.amazonaws.com
       region: eu-west-1
       context: am/openam/samples/docker/images/am-empty
       dockerfile: am/openam/samples/docker/images/am-empty/Dockerfile
       force_tag: true
       tags:
         - "latest"
-        - "${DRONE_COMMIT_SHA}"
+        - "build-${DRONE_BUILD_NUMBER}"
     when:
       event: push
       branch: master
 
   - name: build-base-ds
-    image: openjdk:11
+    image: amidodevelopment/drone-runner-image:latest
```

Comment:
I'm not sure this is good practice. The idea is to build DS with the minimum requirements, which is JRE v11, not a Drone-specific base image which contains many libraries.

Reply:
Ok, wasn't aware of that requirement for DS. Have changed it to use the openjdk image.